requirements-dev.pip

```diff
@@ -1,4 +1,4 @@
-django>=2.2, <2.3
+django>=2.2
```

Comment:
Should this be locked down more specifically?

Reply:
reverted change. Will continue to use 2.2 until the 3.2 readiness is complete.